### ISSUE TITLE
LaVolpe Alphaimage Control

### ISSUE THREAD

**Issue by Cees679** [twinbasic/twinbasic#1907]
In many of my projects I use LaVolpe's Aplhaimage Control. In Twinbasic I can add it, modify it's properties, but both in design time and runtime nothing shows up. Both in Classic VB and VB.net this control is working. Any chance you solve this issue?

**Comment by wqweto** [comment]
Link to OCX you are testing with?

If no OCX is available then send a link to project sources (but w/ binary repro will always be more close to what you are experiencing).

**Comment by fafalone** [comment]
https://www.vbforums.com/showthread.php?630193-vb6-Alpha-Image-Control-v2-Final-Update-(15-Jan-2012) 

Looks like there's unsupported syntax right off the bat; tB can't put UDTs in Variants yet?

**Comment by wqweto** [comment]
OP is experiencing compatibility issues with the compiled version of the control sited on TB forms IMO.

**Comment by fafalone** [comment]
Agreed but particularly since LaVolpe didn't distribute an official OCX build it's worth noting tB currently can't compile one due to missing code support while we're on the topic.
